rust/core-lib/src/view.rs

```diff
@@ -869,8 +882,30 @@ impl View {
         let height = self.line_of_offset(text, text.len()) + 1;
         let plan = RenderPlan::create(height, self.first_line, self.height);
         self.send_update_for_plan(text, client, styles, style_spans, &plan, pristine);
-        if let Some(new_scroll_pos) = self.scroll_to.take() {
-            let (line, col) = self.offset_to_line_col(text, new_scroll_pos);
+        if !self.is_tail_enabled {
```

Comment:
This check makes sure that when user scrolls a tailing file, we dont auto scroll to cursor location. Mimics `tail` command.

Reply:
I suppose it'd be worth adding a comment to the source file about this.

Reply:
Will do.